encoder/vaapiencoder_base.cpp

```diff
@@ -190,10 +190,9 @@ YamiStatus VaapiEncoderBase::setParameters(VideoParamConfigType type, Yami_PTR v
         VideoParamsCommon* common = (VideoParamsCommon*)videoEncParams;
         if (common->size == sizeof(VideoParamsCommon)) {
             PARAMETER_ASSIGN(m_videoParamCommon, *common);
-            //if(m_videoParamCommon.rcParams.bitRate > 0)
-	        // m_videoParamCommon.rcMode = RATE_CONTROL_CBR;
-	     // Only support CQP and CBR mode now
-            if (m_videoParamCommon.rcMode != RATE_CONTROL_CBR)
+            if (m_videoParamCommon.rcMode != RATE_CONTROL_CBR &&
+                m_videoParamCommon.rcMode != RATE_CONTROL_VBR &&
+                m_videoParamCommon.rcMode != RATE_CONTROL_CVBR)
```

Comment:
@lizhong1008 , The other commit 8fe2b89 is for fixing the confusion of mode. This commit is for add VBR and CVBR. I prefer not to squash them.